src/components/CalendarView/DayTileItem.js

```diff
@@ -8,6 +8,10 @@ import { spacing } from "../../theme";
 import { getThemeValue, getLabelTextColor } from "../../utils";
 
 const DayTileItem = styled.article`
+  background-color: ${({ highlighted }) =>
+    highlighted
+      ? getThemeValue("primary", "lightBase")
+      : getThemeValue("transparent")};
```

Comment:
@mlampedx I wanted to write that background setup like this:
`${({ highlighted }) => highlighted && `background-color: ${getThemeValue("primary", "lightBase")}`};`

But `getThemeValue` second function won't be called inside of the inner template string. And if we call it in explicit way `getThemeValue("primary", "lightBase")()` then we'd always get default theme value (that might be hard to detect in future in case another themes would be created)